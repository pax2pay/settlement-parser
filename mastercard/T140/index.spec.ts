import "jest"
// import { T140 } from "./"
// import * as data from "./data"

describe("T140", () => {
	it("test", () => {
		expect(true).toBeTruthy()
	})
	// it("parse testA", () => {
	// 	const outcome = T140.parse(data.testA)
	// 	expect(outcome).toEqual({
	// 		pages: [],
	// 		type: "1IP728010-AA",
	// 	})
	// })
})
