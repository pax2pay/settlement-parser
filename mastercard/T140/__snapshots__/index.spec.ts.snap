// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`T140 parse single 1`] = `
Object {
  "pages": Array [
    Object {
      "class": "acknowledgement",
      "cycle": 3,
      "date": "2023-12-21",
      "member": 33400,
      "run": "2023-12-21",
    },
    Object {
      "class": "notification file",
      "cycle": 3,
      "date": "2023-12-21",
      "file": "001/231221/00000004882/03402",
      "run": "2023-12-21",
    },
    Object {
      "class": "notification summary",
      "content": Array [
        Object {
          "brand": "MCC",
          "currency": "GBP",
          "data": Array [
            Object {
              "amount": 0.73,
              "direction": "original",
              "fee": -0.02,
              "file": "001/231221/00000004882/03402",
              "id": 900,
            },
          ],
          "level": "mbr-to-mbr agreement",
          "member": 33400,
          "total": Object {
            "amount": 0.73,
            "fee": -0.02,
          },
        },
      ],
      "cycle": 3,
      "date": "2023-12-21",
      "run": "2023-12-21",
    },
  ],
}
`;
